mods/cnc/chrome/tooltips.yaml

```diff
@@ -204,6 +204,30 @@ Background@CLIENT_TOOLTIP:
 			Height: 10
 			Font: TinyBold
 
+Background@INGAME_CLIENT_TOOLTIP:
+	Logic: ClientTooltipLogic
+	Background: panel-black
+	Height: 37
```

Comment:
Please make this 35, 37 looks wrong:

35 ![screenshot from 2015-12-10 22 19 59](https://cloud.githubusercontent.com/assets/4331210/11728877/89c243ee-9f8c-11e5-9a73-52147b665346.png) vs. 37 ![screenshot from 2015-12-10 22 20 35](https://cloud.githubusercontent.com/assets/4331210/11728880/8bebf41c-9f8c-11e5-9ddd-a6453dbd4a0e.png)